pkg/apis/kops/cluster.go

```diff
@@ -292,6 +292,10 @@ type EtcdClusterSpec struct {
 	EnableEtcdTLS bool `json:"enableEtcdTLS,omitempty"`
 	// Version is the version of etcd to run i.e. 2.1.2, 3.0.17 etcd
 	Version string `json:"version,omitempty"`
+	// Leader election timeout
+	LeaderElectionTimeout string `json:"leaderElectionTimeout,omitempty"`
```

Comment:
So a sting is probably the wrong type. We have a *int64 on the loadbalancer timeout. Can we please model the same? What is the unit here? Also please follow the go standards for documenting code.

Reply:
can we not use metav1.Duration?

Reply:
Also, are we including the changes in protokube here to use the API values?

Reply:
@gambol99 yes, we will need protokube updates.  I asked to have the base APIs PR'ed first.  

Yes, I agree on the duration.  We should have used that for ELB ;(  We are using it on component config values.

@mschurenko please modify the `string` to `*metav1.Duration` and here is a good guide for go best doc practices https://blog.golang.org/godoc-documenting-go-code

Reply:
It looks like etcd expects just a raw number like `2000` and not `2000ms` for the settings. Do we still want to use `*metav1.Duration` and not `string` or `int64`? I guess we can just convert it to whatever etcd wants in protokube before writing out the manifest?